src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.nuspec

```diff
@@ -1,17 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
-    <id>$id$</id>
-    <version>$version$</version>
-    <authors>$authors$</authors>
-    <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>$licenseUrl$</licenseUrl>
-    <projectUrl>$projectUrl$</projectUrl>
-    <iconUrl>$iconUrl$</iconUrl>
-    <description>$description$</description>
-    <copyright>$copyright$</copyright>
-    <tags>$tags$</tags>
-    <repository type="git" url="$repositoryUrl$" commit="$repositoryCommit$" />
+    $CommonMetadataElements$
```

Comment:
Just to confirm: The Arcade helpers i.e. the `InitializeStandardNuspecProperties` target does the `$CommonMetadataElements$` substitution?

Reply:
Yup: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L102